scraper/main.py

```diff
@@ -5,25 +5,56 @@
 import re
 from bs4 import BeautifulSoup as bs
 import html5lib
-from datetime import datetime
+from datetime import datetime, date
 import time
-from dotenv import load_dotenv
-import schedule
-from multiprocessing import Process
 import sys
+import json
+
+def scrape_stock(target):
+
+    data = {}
 
-def scrape_stock():
-    stock_symbol = (input('Input stock symbol: ').upper())
-    print(f'{stock_symbol} selected')
+    data['scrape time'] = (datetime.utcnow()).strftime("%H:%M:%S")
+    data['today'] = (datetime.today()).strftime("%B %d, %Y")
 
-    url = f'https://finance.yahoo.com/quote/{stock_symbol}?p={stock_symbol}&.tsrc=fin-srch'
+    url = f'https://finance.yahoo.com/quote/{target}?p={target}&.tsrc=fin-srch'
 
     r = requests.get(url)
     page_content = bs(r.content, features='html5lib')
-    data_table = page_content.find("div", attrs={"id": "quote-summary"})
 
-    open_value = data_table.find('td', attrs={'data-test': 'OPEN-value'})
-    parse_open = (open_value.find('span')).string
-    print(f'Opened at ${parse_open}')
+    # Check for valid symbol
+    try:
+        data_table = page_content.find("div", attrs={"id": "quote-summary"})
+    except:
+        return False
+
+    # Parse various containers
+    parse_open = data_table.find('td', attrs={'data-test': 'OPEN-value'})
+    parse_points_close = page_content.find('div', attrs={'class' : 'D(ib) Mend(20px)'})
+    parse_cap = data_table.find('td', attrs={'data-test': 'MARKET_CAP-value'})
+
+    # Further parse containers and extract strings
+    data['stock name'] = (page_content.find('h1', attrs={'data-reactid': '7'})).string
+    data['points change'] = (parse_points_close.contents[1]).string
+    data['open'] = (parse_open.find('span')).string
+    data['close'] = (parse_points_close.contents[0]).string
```

Comment:
From what I see, `data['close']` represents the current stock price. Maybe we can give it a more descriptive name.

Reply:
You're right, I'll get that fixed. Also need to rebuild the error handling since it looks like I bjorked it while converting to using json.